src/main/resources/db/migration/V006__Add_envelope_events_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE envelope_events
+(
+    id          BIGSERIAL    PRIMARY KEY,
+    envelope_id UUID         NOT NULL REFERENCES envelopes ON DELETE CASCADE,
+    created_at  TIMESTAMP    NOT NULL,
+    type        VARCHAR(100) NOT NULL,
+    notes       TEXT         NULL
```

Comment:
worth renaming this to error_description or something else

Reply:
Let's stick to existing name from previous model, in case we want to put any notes on successful events.
We can rename it later.